addons/sale_loyalty/models/sale_order.py

```diff
@@ -75,6 +77,20 @@ def _compute_loyalty_data(self):
                 'cost': loyalty_history_data_per_order[order.id]['total_cost'],
             }
 
+    def _compute_gift_card_count(self):
+        gift_card_data = dict(
+            self.env['loyalty.card']._read_group(
+                domain=[
+                    ('order_id', 'in', self.ids),
+                    ('program_type', '=', 'gift_card'),
+                ],
+                groupby=['order_id'],
+                aggregates=['order_id:count'],
```

Comment:
From what i see in the code, the aggregate `__count` seems to be frequently used for that situation when the aggregate is a count of the `groupby` field, maybe you could use it too here :thinking:

Reply:
Thanks for pointing this out. It's working as intended with the `__count`.